repos/system_upgrade/el7toel8/actors/selinux/selinuxprepare/tests/component_test.py

```diff
@@ -0,0 +1,101 @@
+import os
+
+import pytest
+
+from leapp.snactor.fixture import current_actor_context
+from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom
+from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.reporting import Report
+
+enabled = False
+
+test_modules = [
+    ["400", "mock1"],
+    ["99", "mock1"],
+    ["300", "mock1"],
+    ["400", "mock2"],
+    ["999", "mock3"]
+]
+
+semanage_commands = [
+    ['fcontext', '-t', 'httpd_sys_content_t', '"/web(/.*)?"'],
+    ['fcontext', '-t', 'ganesha_var_run_t', '"/ganesha(/.*)?"'],
+    ['port', '-t', 'http_port_t', '-p', 'udp', '81'],
+    ['permissive', 'abrt_t']
+]
+
+# save value of semodule -lfull for comparison
+semodule_lfull = ""
+semanage_export = ""
+
+if enabled:
+    try:
+        semodule = run(["semodule", "-lfull"], split=False)
+        semodule_lfull = semodule.get("stdout", "")
+        semanage = run(["semanage", "export"], split=False)
+        semanage_export = semanage.get("stdout", "")
+    except CalledProcessError as e:
+        api.current_logger().warning("Error listing SELinux customizations: %s", str(e.stderr))
+
+testmoduledir = os.path.join(os.getcwd(), "tests/mock_modules/")
+
+
+def setup():
```

Comment:
Not skipped as other tests. Wondering how to do this straight (without blowing up a system during setup/teardown for a skipped test and still being able to run component tests automatically). An open question for the whole leapp team I guess :)

Reply:
Thanks for the changes.